ddapm_test_agent/agent.py

```diff
@@ -722,6 +724,13 @@ def main(args: Optional[List[str]] = None) -> None:
         print(_get_version())
         sys.exit(0)
 
+    apm_sock: Optional[socket.socket]
+    if parsed_args.trace_uds_socket is not None:
+        apm_sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+        apm_sock.bind(parsed_args.trace_uds_socket)
+        os.chmod(parsed_args.trace_uds_socket, 0o777)
```

Comment:
722 is probably fine since that is what the agent does but I understand wanting to be permissive

Reply:
good call, let's just mirror what the agent does